ms/MeasurementSets/MeasurementSet.cc

```diff
@@ -308,11 +312,15 @@ MeasurementSet::operator=(const MeasurementSet &other)
 	mrsDebugLevel_p = other.mrsDebugLevel_p;
 	memoryResidentSubtables_p = other.memoryResidentSubtables_p;
 
-	copySubtables (other);
+        if (! (isNull()  ||  other.isNull())) {
```

Comment:
Why do you need the `isNull()` check? It looks like checking `if (!other.isNull())` should be enough here.